src/VirtualClient/VirtualClient.Main/profiles/PERF-MONGODB.json

```diff
@@ -0,0 +1,66 @@
+{
+  "Description": "MongoDB Workload",
+  "Metadata": {
+    "RecommendedMinimumExecutionTime": "00:05:00",
+    "SupportedPlatforms": "linux-x64,linux-arm64,win-x64,win-arm64",
+    "SupportedOperatingSystems": "CBL-Mariner,CentOS,Debian,RedHat,Suse,Ubuntu,Windows"
+  },
+  "Parameters": {},
+  "Actions": [
+    {
+      "Type": "MongoDBExecutor",
+      "Parameters": {
+        "Scenario": "",
+        "PackageName": "mongodb",
+        "YCSBPackageName": "ycsb",
+        "JdkPackageName": "javadevelopmentkit",
+        "Tags": ""
+      }
+    }
+  ],
+  "Dependencies": [
+    {
+      "Type": "DependencyPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallJDKPackage",
+        "BlobContainer": "packages",
+        "BlobName": "microsoft-jdk-11.0.19.zip",
+        "PackageName": "javadevelopmentkit",
+        "Extract": true
+      }
+    },
+    {
+      "Type": "DependencyPackageInstallation",
+      "Parameters": {
+        "Scenario": "InstallYCSBPackage",
+        "BlobContainer": "packages",
+        "BlobName": "ycsb-0.5.0.zip",
```

Comment:
I saw these three benchmarking tool in MongoDB's own blog. https://www.mongodb.com/blog/post/performance-best-practices-benchmarking
![image](https://github.com/microsoft/VirtualClient/assets/31965446/0bafed8b-33b4-4ea7-acf5-07b6affb7027)
Have you research the other 2?